src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1684,10 +1681,16 @@ public static bool TryConvertTo<T>(object valueToConvert, out T result)
         /// <returns>false for conversion failure, true for success</returns>
         public static bool TryConvertTo<T>(object valueToConvert, IFormatProvider formatProvider, out T result)
         {
-            result = default(T);
+            result = default;
             try
```

Comment:
The `try/catch` can be removed, as the exceptions are taken care of in `TryConvertTo(valueToConvert, typeof(T), formatProvider, out var res)`.